op-acceptance-tests/tests/base/chain_test.go

```diff
@@ -0,0 +1,48 @@
+package base
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/ethereum-optimism/optimism/devnet-sdk/system"
+	"github.com/ethereum-optimism/optimism/devnet-sdk/testing/systest"
+	"github.com/ethereum-optimism/optimism/op-service/testlog"
+	"github.com/ethereum/go-ethereum/log"
+)
+
+// TestChainFork checks that the chain does not fork.
+func TestChainFork(t *testing.T) {
+	chainIdx := uint64(0)
+
+	systest.SystemTest(t,
+		chainForkTestScenario(chainIdx),
+	)
+}
+
+func chainForkTestScenario(chainIdx uint64) systest.SystemTestFunc {
+	return func(t systest.T, sys system.System) {
+		const numCyclesToWait = 5
+
+		logger := testlog.Logger(t, log.LevelInfo)
+		logger.Info("Started test")
+
+		chain := sys.L2s()[chainIdx]
+
+		// Check for a chain fork
+		logger.Info("Checking for chain fork")
+		laterCheck, err := systest.CheckForChainFork(t.Context(), chain, logger)
```

Comment:
This only does something interesting if there are multiple nodes running in this devnet (for each chain). Let's bear that in mind when we are deciding when to run this test -- i.e. are we running it on a simple devnet with a single sequencer? There can't be a chain split on such a network.